src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddFileMetadataToDepositSpec.scala

```diff
@@ -160,24 +219,25 @@ class AddFileMetadataToDepositSpec extends UnitSpec with BeforeAndAfter {
       case Failure(ActionException(_, message, _)) =>
         message should {
           include("Values found for these columns: [SF_COLLECTION]") and
-            include("these columns should be empty because there are no audio/video")
+            include("these columns should be empty because there are no audio/video files found in this dataset")
```

Comment:
Line 202 says "should succeed" while 219 checks for a failure. So 202 should sound more like "should complain about no AV files while SF_COLLECTION is provided" or "should reject ... because ...".